README.md

```diff
@@ -12,7 +12,7 @@ This is the first project at Microverse second week with coding partner and my p
 To get a local copy up and running follow these simple steps.
 
 1. Clone the Repo or Download the Zip file (git clone git@github.com:aster-alemu/Mobile-Version.git)
-2. Run the live server or simply open index.html with your browser on local machine.
+2. Run the live server using this link: https://aster-alemu.github.io/Mobile-Version/
```

Comment:
- [ ] [OPTIONAL] The live demo link should be under a header of its own and not under Getting Started. This is an example below. Please kindly make changes:

```
## Live Demo

[Live-Demo](https://kelomo2502.github.io/portfolio/)
```